keyboards/gray_studio/cod67/config.h

```diff
@@ -49,135 +49,26 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 /* COL2ROW, ROW2COL*/
 #define DIODE_DIRECTION COL2ROW
 
+/* D4 is not a PWM pin so the board only supports toggle */
```

Comment:
This actually is not true anymore, the Backlight feature can do [hardware timer assisted software PWM](https://docs.qmk.fm/#/feature_backlight?id=timer-assisted-pwm-implementation), which is pretty solid 👍